0002-capture-matching-next-steps.md

```diff
@@ -0,0 +1,115 @@
+
+* {e.g., improvement of quality attribute satisfaction, follow-up decisions required, …}
+* …
+
+### Negative Consequences <!-- optional -->
+
+* {e.g., compromising quality attribute, follow-up decisions required, …}
+* …
+-->
+
+## Pros and Cons of the Options
+
+### 1. Implement mobile application side image collection improvements, including enforced phone rotation, enforced compass bearing, and perhaps others.
+
+Currently, images captured on the mobile application do not have specific criteria guiding capture.  Thus, they exist at a variety of camera rotation angles and compass bearings, even for the same tree being repeat captured.  This requires a manual operator to consider images of the same tree that almost always exist at different rotations and focal distances, greatly enhancing the difficulty of an already challenging manual task.
+
+Constraints can be implemented on the mobile side to ameliorate this situation.  For instance, camera rotation around the x axis could be constrained to 0, 45, and 90 degrees.  Additionally, compass bearing could be constrained such that captured photos are always north facing.  Implementing a constraint on focal distance is also possible, as has been identified in other apps.  
+
+* Good, because the images captured will have more constitant parameters.
+* Good, because this only affect the mobile application and does not require data research or algorithm development.
+* Bad, because the mobile application will become more complex for our low technical literacy users, and we will have to invest careful UX/UI development time to make these features easily usable.
+
+### 2. Research feasibility of leveraging error autocorrelation between GPS measurements to improve selection of candidates for location based matching of captures.
+
+Preliminary research has shown that GPS measurements between trees, and along the path of tree trackers traveling between trees, have errors against true position that are highly autocorrelated.  This means that while the absolute position of a give measurement can be 3-10m off using L2 GPS technology, the direction of travel between measurements is relatively stable across site visits.  This presents opportunities for leveraging this similarity to identify captures in the vicinity of a capture from a previous site visit that are more likely to be repeat visits to the same tree, which can be used to present more likely candidates during the manual matching process.
+
+* Good, because this does not affect the mobile users
+* Good, because this feature uses data we are already collecting
+* Good, because some preliminary research has already been done (@ZavenArra) that shows potential for this solution to work.
+* Bad, because this feature requires advanced skills in GIS analysis and algorithm development.
+
+### 3. Create experimental manual capture matching UI that offers manual spatial analysis relevant to disambiguating matching captures.
```

Comment:
I think this might be the best bet because I think it would be easier to train admin users as opposed to making the application more complex. Other options seem to require a lot more 'resources'.

Reply:
Plus over time the data collected could likely be used to assess some of the automated solutions, which seems to me is better than committing to automation a priori and using biased or fabricated datasets for it.

Reply:
I agree that this option feels like it has a lot of potential for improvements in the short term. I think we could prototype a UI fairly quickly to collect feedback on its utility and simplicity for users. If we do go down the automated route, we may still want a UI to show the correlation and help spot errors.

Reply:
@camwebb The implementation of your suggestion here is mailing option 1, enforcing orientations in the camera - or more specifically compass heading.  Are you voting for compass heading enforcement in the image capture process as the low hanging fruit?

Reply:
@ZavenArra No, I'm not suggesting enforcing anything. Simply giving the Admin panel user easy to assimilate info on how the photo was taken.

Reply:
I see, so simply _reporting_ the compass bearing and smartphone rotation matrix to the admin panel user.  We have facilities to collect this information, so I think this is a great suggestion on how to start making use of it.   There are a few other parameters we collect, such as step count since last capture, that could also be reported.